addons/mail/static/src/components/attachment_viewer/attachment_viewer.xml

```diff
@@ -5,24 +5,25 @@
         <t t-if="attachmentViewer">
             <div class="o_AttachmentViewer flex-column align-items-center d-flex w-100 h-100" t-attf-class="{{ className }}" t-on-click="attachmentViewer.onClick" t-on-keydown="_onKeydown" tabindex="0" t-ref="root">
                 <div class="o_AttachmentViewer_header d-flex w-100 bg-black-75 text-400" t-on-click="attachmentViewer.onClickHeader">
-                    <t t-if="attachmentViewer.attachment.isViewable">
+                    <t t-set="attachment" t-value="attachmentViewer.attachment"/>
```

Comment:
Don't add this `t-set`, this is introducing a lot of unnecessary diff.

Reply:
In documents we may have documents that have no attachment (youtube links for example) as such it was necessary to avoid rewriting all lines that use `attachmentViewer.attachment`.
This is the same reasoning behind the isMultiAttachment field.
Another way would be to make a computed field such as `attachmentOrDocument` but it does not make much sense when documents is not installed.
What do you think?